tests/test_interface.rb

```diff
@@ -40,7 +40,8 @@ class TestInterface < CiscoTestCase
   DEFAULT_IF_IP_PROXY_ARP = false
   DEFAULT_IF_IP_REDIRECTS = true
   DEFAULT_IF_VRF = ''
-  IF_DESCRIPTION_SIZE = 243 # SIZE = VSH Max 255 - "description " keyword
+  IF_DESCRIPTION_VALID_SIZE = 68 # SIZE = VSH Max 80 - "description " keyword
+  IF_DESCRIPTION_INVALID_SIZE = 255
```

Comment:
Just a thought: The description test is a carry-over from the old OneP code. I'm of the opinion that a lot of these out-of-range tests are outside the scope of our APIs -- why should my setter care that the max char limit is 80 on one platform and 255 on another? ie. attempt to set the manifest's requested state and let the platform reject it.  

IOW, my suggestion is to just remove these tests.